pennylane/concurrency/executors/native/multiproc.py

```diff
@@ -43,7 +43,7 @@ class MPPoolExec(PyNativeExec):
 
     @classmethod
     def _exec_backend(cls):
-        return Pool
+        return multiprocessing.get_context("spawn").Pool
```

Comment:
```suggestion
        return get_context("spawn").Pool
```

Just a little tidier